### PR TITLE
Update aqbanking+gwenyhwfar versions

### PR DESCRIPTION
**Pull request by cstim**

### CODE REVIEW

**Comment by fellen** [comment]
Interesting, there is nothing about in https://www.aquamaniac.de/rdm/projects/aqbanking/news.

File: gnucash.modules [hunk 1]
```diff
@@ -147,7 +147,7 @@
```
Comment (fellen) [inline gnucash.modules]: libchipcard should be 221/libchipcard-5.1.4rc1.tar.gz

Comment (cstim) [reply gnucash.modules]: yes, I changed the commit accordingly.

**Comment by jralls** [comment]
The filename for libchipcard is wrong, it should be `libchipcard-5.1.4rc1.tar.gz` instead of `libchipcard-5.1.4beta.tar.gz`. No matter, that's an easy fix. 
There's also this error at the end of configure:
```
configure: WARNING: unrecognized options: --disable-Werror
ln: failed to create symbolic link 'libchipcard5/chipcard/cards/hiinsurancedata.h': File exists
ln: failed to create symbolic link 'libchipcard5/chipcard/cards/hipersonaldata.h': File exists
```
I think it's harmless: The files exist so must have been copied/linked (the same thing as far as MSYS2 is concerned, it doesn't know about NT symlinks) already.

There's a libtool error that breaks installing with DESTDIR in an MSYS2 shell but it works OK from powershell so we'll ignore that too.

Pushed with the filename and version fix.

**Pull request closed without merging**